charts/stable/synapse/questions.yaml

```diff
@@ -342,19 +348,6 @@ questions:
                   type: boolean
                   default: true
 
-  - variable: coturn
-    group: "Container Configuration"
-    label: "Coturn Configuration"
-    schema:
-      additional_attrs: true
-      type: dict
-      attrs:
-        - variable: enabled
-          label: "Enable Coturn"
-          schema:
-            type: boolean
-            default: false
-
```

Comment:
please don't delete things...
It doesn't make it more mergeable. Less changes improves the chance of anything being merged.

I already don't have much time to go over this, if you start deleting things I build, I can assure you it is going to be oktober.

Just keep the changes to a minimum required for it to function.